src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submissions.cljs

```diff
@@ -0,0 +1,119 @@
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.paths :as paths]
+    [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+(defn submission-running? [submission]
+  (let [status (get submission :status)]
+    (= status "Running")))
+
+(defn- create-mock-submissions-list [workspace]
+  (map
+    (fn [i]
+      {:workspaceName {:namespace (get workspace "namespace")
+                       :name (get workspace "name")}
+       :methodConfigurationNamespace "my_test_configs"
+       :submissionDate (str "2015-" (rand-nth (range 1 12)) "-"
+                         (rand-nth (range 1 30)) "T19:08:53.027Z")
+       :submissionId "46bfd579-b1d7-4f92-aab0-e44dd092b52a"
+       :notstarted []
+       :workflows [{:messages []
+                    :workspaceName {:namespace (get workspace "namespace")
+                                    :name (get workspace "name")}
+                    :statusLastChangedDate (str "2015-" (rand-nth (range 1 12))
+                                              "-"(rand-nth (range 1 30))
+                                              "T19:08:53.027Z")
+                    :workflowEntity {:entityType "sample"
+                                     :entityName "sample_01"}
+                    :status "Succeeded"
+                    :workflowId "97adf170-ee40-40a5-9539-76b72802e124"}]
+       :methodConfigurationName (str "test_config"(inc i))
+       :status (rand-nth ["Running" "Failed" "Completed" "Queued"])
+       :submissionEntity {:entityType "sample"
+                          :entityName (str "sample_"(inc i))}
+       :submitter "abaumann@broadinstitute.org"})
+       (range (rand-int 50))))
+
+
+(defn render-running-loaded-single-submission-summary [submission]
+  [:div {}
+   [:hr]
+    [:div {}
+    [:div {:style {:float "left" :width "50%" :height "15%"}}
+     [:div {:style {:marginLeft "1em"} }
+      (get submission :methodConfigurationName)
+      [:br]
+      (str "Status : "
+        (get submission :status) " ; Started At : " (get submission :submissionDate) )
+      [:br]
+      (let [submissionEntity (get submission :submissionEntity)
+            entityName (get submissionEntity :entityName)
+            entityType (get submissionEntity :entityType)]
+        (str "Entities : " entityType "/" entityName ))]]
+    [:div {:style {:float "right" :width "50%" :height "15%"}}
+       [comps/Button
+          {:color "#BBB"
+           :text "Details..."
+           :onClick (fn [e]
+                   (js/alert "take me to the details please"))}]
+       [:br]
+       [comps/Button
+        {:color "Red"
+         :text "Abort"
+         :onClick
+         (fn [] (utils/rlog "TODO : abort the submission here!"))
+         }]
+        [:input {:type "checkbox"}]
+        "I really mean it!"]]])
```

Comment:
By " this design " I assume you refer to the pixelmedia design?

I purposely coded with the intent to imitate the design as I saw it.

My opinion on the pixelmedia (whose link is https://extranet.pixelmedia.com/pSecured/pClients/pBuild/BDI405/files/Broad%20Wireframes16_final/index.html#p=_workspace_monitor_methods ) is that the design is okay.

The moving of the "Details..." button to the left seems good in my opinion.  Someone wanting to see details might inadvertently hit "Abort"!

Based on my prior experience the usage of a "confirmation checkbox" seems somehow an "outlier" a  "non-traditional" UI choice.  Maybe the choice was made specifically to BE an outlier and specifically to be used to cause people to take notice when trying to abort?  Maybe for the abort button there should be *both* a confirmation checkbox AND a confirmation dialog?  Maybe the "confirmation checkbox" was put in the pixel media page under the assumption that it would be interpreted as you interpret it now Matt as a "confirmation dialog" page?  Maybe someone thoughtlessly put a "confirmation checkbox"?  I speculate only.  Regarding the "Confirmation Checkbox" I don't have a strong opinion either way, but I have just offered some speculative thoughts.